workspace-loader/src/index.ts

```diff
@@ -139,26 +139,29 @@ export class WorkspaceLoader {
         window.parent.postMessage("show-ide", "*");
     }
 
-    load(): Promise<void> {
+    async load(): Promise<void> {
         const workspaceKey = this.getWorkspaceKey();
 
         if (!workspaceKey || workspaceKey === "") {
             console.error("Workspace is not defined");
             return;
         }
 
-        return this.getWorkspace(workspaceKey)
-            .then((workspace) => {
-                this.workspace = workspace;
-                return this.handleWorkspace();
-            })
-            .then(() => this.openIDE())
-            .catch(err => {
+        try {
+            this.workspace = await this.getWorkspace(workspaceKey);
+            await this.handleWorkspace()
+            await this.openIDE();
+        }
+        catch(err) {
+            if(err) {
                 console.error(err);
                 this.loader.error(err);
-                this.loader.hideLoader();
-                this.loader.showReload();
-            });
+            } else {
+                this.loader.error('Unknown error are happened, try to reload page');
```

Comment:
Unknown error are happened -> could be
- Unknown error occurred, ..
- Unknown error has happened